scrapy/linkextractors/lxmlhtml.py

```diff
@@ -171,8 +171,10 @@ def __init__(
         allow_domains: str | Iterable[str] = (),
         deny_domains: str | Iterable[str] = (),
         restrict_xpaths: str | Iterable[str] = (),
-        tags: str | Iterable[str] = ("a", "area"),
-        attrs: str | Iterable[str] = ("href",),
+        tags: str | Iterable[str] | bool = ("a", "area"),
+        attrs: str | Iterable[str] | bool = ("href",),
+        deny_tags: str | Iterable[str] = (),
+        deny_attrs: str | Iterable[str] = (),
```

Comment:
I think we might want to set all 4 parameters to `None` at this point, to easily detect when they are set (even to their intended default value) vs when they are not passed.

Then, later:
- If an “allow parameter” is used, its deny counterpart must be `None`, else a `ValueError` should be raised about it.
- If both are `None`, we assume the allow parameter with its default value (as was set in the function signature here).